### ISSUE TITLE
Questionnaire

### ISSUE THREAD

**Issue by mayacakmak** [mayacakmak/se2#10]
First version completed by Kavi, some formatting edits by Maya.

Revising list of questions in the [Google doc](https://docs.google.com/document/d/1nDIg68uMvRvAqv7C7-pfBeU8chcG3bRhDE97Uh8wM04/edit#).

- [x] Finalize questions 
- [x]  Finalize question formatting throughout
- [x] Add code to send the form data to the database to attach to the current users data.
- [x] Red asteriks on required questions
- [x] Is the data remembered if the user goes back and forth?

**Comment by kavidey** [comment]
Data should be remembered if users go back and forth (behind the scenes, going back and forth is just hiding and revealing groups of elements using JQuery, so it keeps their state the same). As long as the user doesn't reload, everything should be saved.

I just added the red asterisk:
![image](https://user-images.githubusercontent.com/16945761/90320723-74bcb000-def8-11ea-9a1f-683f713d0ec1.png)

The code for logging to firebase is also finished. Currently, it is just logging the section and number of the question instead of the whole name. Each questionnaire is logged under `/users/[uid]/sessions/[sid]/questionnaires` and contains a timestamp, so in the case that a user filled it out twice on different days or something, we would be able to differentiate them.
![image](https://user-images.githubusercontent.com/16945761/90320757-a46bb800-def8-11ea-9b54-f815e865bf1b.png)

I also added some code to display the user ID for MTurk payment (I'm still working on the other parts of MTurk integration):
![image](https://user-images.githubusercontent.com/16945761/90320839-396eb100-def9-11ea-94a4-775d8b4a1634.png)
